frontend/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -44,7 +48,7 @@ export const RadioGroup = <T extends string | number>({
 		)
 	})
 	return (
-		<div style={style} className={styles.radioGroupWrapper}>
+		<div style={wrapperStyle} className={styles.radioGroupWrapper}>
```

Comment:
ended up introducing two new style props for wrapper and the "child" label.
That should make things clear